CentralManagement/ChocoCCM Command Reference/GetCCMOutdatedSoftwareReportDetail.md

```diff
@@ -0,0 +1,63 @@
+# Get-CCMOutdatedSoftwareReportDetail
+
+<!-- This documentation is automatically generated from /Get-CCMOutdatedSoftwareReportDetail.ps1 using GenerateDocs.ps1. Contributions are welcome at the original location(s). -->
+
+View detailed information about an Outdated Software Report
+
+## Syntax
+
+~~~powershell
+Get-CCMOutdatedSoftwareReportDetail `
+  -Report <String> [<CommonParameters>]
+~~~
```

Comment:
This command should just be part of `Get-CcmOutdatedSoftwareReport` IMO. This command name is long and unwieldy, and it doesn't need to be a separate command.